Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -4768,6 +4768,7 @@ def gen2024HiMix(fragment,howMuch):
 defaultDataSets['Run4D96']='CMSSW_13_1_0_pre1-130X_mcRun4_realistic_v2_2026D96noPU-v'
 defaultDataSets['Run4D98']='CMSSW_13_2_0_pre1-131X_mcRun4_realistic_v5_2026D98noPU-v'
 defaultDataSets['Run4D110']='CMSSW_15_1_0_pre5-150X_mcRun4_realistic_v1_STD_RegeneratedGS_Run4D110_noPU-v'
+defaultDataSets['Run4D121']=''
```

Comment:
@cms-sw/pdmv-l2 is there anything that we can put here? I could not find any D121 samples in DAS.

Reply:
Not really. We can produce them. 

In general I think that if we want to move definetly to D121, we'd need a validation campaign.

Reply:
Maybe @AdrianoDee could help to submit minbias relvals with D121 with existing release (e.g. 16_0_0_pre1 or pre2). Then, within the next few days, when dataset is available, this PR can be updated.

Reply:
By the way, not sure how short matrix for PR test will work for now, as it will try to run D121+PU workflow, right?

Reply:
Including pre2 relvals for this geometry in this PR would be ideal.

Indeed, a few tests may fail, but I think most do not include PU.

Reply:
I've sent a set of D121 relvals [here](https://cms-pdmv-prod.web.cern.ch/relval/relvals?ticket=CMSSW_16_0_0_pre2__Phase2_Run4D121_noPU_Reference-00001&shown=2099199&page=0&limit=50). Most of them are already done. So there you can use `CMSSW_16_0_0_pre2-150X_mcRun4_realistic_v1_STD_RegeneratedGS_Run4D121_noPU-v`.

For D121 validation itself, since it should be minor, I would proceed by just having both D110 and D121 samples produced for the incoming `16_0_0_pre3` campaign and have the validators inspect them there.

Reply:
Thanks @AdrianoDee ! Your proposal for validation sounds good to me.